### PR TITLE
Case 21899: New Audio Mute States

### PR DESCRIPTION
**Pull request by wayne-chen**
https://highfidelity.manuscript.com/f/cases/21899/New-Mic-Bar

# TEST PLAN
- Launch Interface in Desktop mode
- Should see a new mic bar with vertical audio level meter as noted in figma document replacing the horizontal bar
- Clicking the mic bar will still toggle mute unless push-to-talk is enabled
- bubble icon next to the mic bar should toggle privacy shield upon click
- "MUTED" should show up when you speak into your microphone while you are muted
- Enable push-to-talk in `Settings > Audio`
- When you are not pushing to talk (using T keypress), the audio level meter should not move
- When you are pushing to talk, audio level meter should move
- Switch to HMD mode
- The mic bar should be horizontal (old layout) and should function the same way as what's in the current master branch
- "MUTED" should show up close to the bottom center of your screen when you speak into your microphone while you are muted
- Open the tablet
- The mic bar with horizontal audio level meter should be replaced with the new mic bar with the same functionality as what's in desktop mode
- open the `Audio` app
- Switch the "Warn when muted in HMD" setting to off
- Talk into the microphone while you are muted
- "MUTED" text should not show up in the bottom center of the screen

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17710/HighFidelity-Beta-PR15048-fbed739.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17710/HighFidelity-Quest-Beta-PR15048-fbed739.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17710/HighFidelity-Beta-PR15048-fbed739.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17710/HighFidelity-Beta-Interface-PR15048-fbed739.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17710/HighFidelity-Beta-PR15048-fbed739.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17710/HighFidelity-Beta-Interface-PR15048-fbed739.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17760/HighFidelity-Beta-PR15048-b18eda7.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17760/HighFidelity-Quest-Beta-PR15048-b18eda7.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17760/HighFidelity-Beta-PR15048-b18eda7.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17760/HighFidelity-Beta-Interface-PR15048-b18eda7.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17760/HighFidelity-Beta-PR15048-b18eda7.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17760/HighFidelity-Beta-Interface-PR15048-b18eda7.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17765/HighFidelity-Beta-PR15048-fa29283.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17765/HighFidelity-Quest-Beta-PR15048-fa29283.apk)

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17780/HighFidelity-Beta-PR15048-b361916.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17780/HighFidelity-Quest-Beta-PR15048-b361916.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17780/HighFidelity-Beta-PR15048-b361916.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17780/HighFidelity-Beta-Interface-PR15048-b361916.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17797/HighFidelity-Beta-PR15048-2e14db5.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17797/HighFidelity-Quest-Beta-PR15048-2e14db5.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17797/HighFidelity-Beta-PR15048-2e14db5.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17797/HighFidelity-Beta-Interface-PR15048-2e14db5.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18079/HighFidelity-Beta-PR15048-2681536.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18079/HighFidelity-Quest-Beta-PR15048-2681536.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18079/HighFidelity-Beta-PR15048-2681536.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18079/HighFidelity-Beta-Interface-PR15048-2681536.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18142/HighFidelity-Beta-PR15048-8196da5.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18142/HighFidelity-Beta-Interface-PR15048-8196da5.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18142/HighFidelity-Beta-PR15048-8196da5.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18142/HighFidelity-Quest-Beta-PR15048-8196da5.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18142/HighFidelity-Beta-PR15048-8196da5.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18142/HighFidelity-Beta-Interface-PR15048-8196da5.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18163/HighFidelity-Beta-PR15048-ea1adac.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18163/HighFidelity-Beta-Interface-PR15048-ea1adac.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18163/HighFidelity-Beta-PR15048-ea1adac.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18163/HighFidelity-Quest-Beta-PR15048-ea1adac.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18163/HighFidelity-Beta-PR15048-ea1adac.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18163/HighFidelity-Beta-Interface-PR15048-ea1adac.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18185/HighFidelity-Beta-PR15048-ffabb14.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18185/HighFidelity-Quest-Beta-PR15048-ffabb14.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18185/HighFidelity-Beta-PR15048-ffabb14.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18185/HighFidelity-Beta-Interface-PR15048-ffabb14.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18185/HighFidelity-Beta-PR15048-ffabb14.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18185/HighFidelity-Beta-Interface-PR15048-ffabb14.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18244/HighFidelity-Beta-PR15048-0a13479.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18244/HighFidelity-Beta-Interface-PR15048-0a13479.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18244/HighFidelity-Beta-PR15048-0a13479.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18244/HighFidelity-Quest-Beta-PR15048-0a13479.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18244/HighFidelity-Beta-PR15048-0a13479.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18244/HighFidelity-Beta-Interface-PR15048-0a13479.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18247/HighFidelity-Beta-PR15048-ff3bb5c.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18247/HighFidelity-Beta-Interface-PR15048-ff3bb5c.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18247/HighFidelity-Beta-PR15048-ff3bb5c.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18247/HighFidelity-Quest-Beta-PR15048-ff3bb5c.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18247/HighFidelity-Beta-PR15048-ff3bb5c.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18247/HighFidelity-Beta-Interface-PR15048-ff3bb5c.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18252/HighFidelity-Beta-PR15048-cbff194.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18252/HighFidelity-Quest-Beta-PR15048-cbff194.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18252/HighFidelity-Beta-PR15048-cbff194.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18252/HighFidelity-Beta-Interface-PR15048-cbff194.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18252/HighFidelity-Beta-PR15048-cbff194.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18252/HighFidelity-Beta-Interface-PR15048-cbff194.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18269/HighFidelity-Beta-PR15048-cbfdb67.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18269/HighFidelity-Beta-Interface-PR15048-cbfdb67.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18269/HighFidelity-Beta-PR15048-cbfdb67.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18269/HighFidelity-Beta-Interface-PR15048-cbfdb67.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18269/HighFidelity-Beta-PR15048-cbfdb67.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18269/HighFidelity-Quest-Beta-PR15048-cbfdb67.apk)

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18287/HighFidelity-Beta-PR15048-c512f3f.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18287/HighFidelity-Quest-Beta-PR15048-c512f3f.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18287/HighFidelity-Beta-PR15048-c512f3f.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18287/HighFidelity-Beta-Interface-PR15048-c512f3f.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18287/HighFidelity-Beta-PR15048-c512f3f.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18287/HighFidelity-Beta-Interface-PR15048-c512f3f.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18289/HighFidelity-Beta-PR15048-a583a41.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18289/HighFidelity-Beta-Interface-PR15048-a583a41.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18289/HighFidelity-Beta-PR15048-a583a41.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18289/HighFidelity-Beta-Interface-PR15048-a583a41.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18289/HighFidelity-Beta-PR15048-a583a41.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18289/HighFidelity-Quest-Beta-PR15048-a583a41.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18292/HighFidelity-Beta-PR15048-fe016f9.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18292/HighFidelity-Beta-Interface-PR15048-fe016f9.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18292/HighFidelity-Beta-PR15048-fe016f9.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18292/HighFidelity-Quest-Beta-PR15048-fe016f9.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18292/HighFidelity-Beta-PR15048-fe016f9.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18292/HighFidelity-Beta-Interface-PR15048-fe016f9.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18294/HighFidelity-Beta-PR15048-3858efd.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18294/HighFidelity-Quest-Beta-PR15048-3858efd.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18294/HighFidelity-Beta-PR15048-3858efd.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18294/HighFidelity-Beta-Interface-PR15048-3858efd.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18294/HighFidelity-Beta-PR15048-3858efd.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18294/HighFidelity-Beta-Interface-PR15048-3858efd.exe

**Comment by wayne-chen** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18348/HighFidelity-Beta-PR15048-3858efd.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18348/HighFidelity-Beta-Interface-PR15048-3858efd.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18348/HighFidelity-Beta-PR15048-3858efd.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18348/HighFidelity-Beta-Interface-PR15048-3858efd.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18348/HighFidelity-Beta-PR15048-3858efd.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18348/HighFidelity-Quest-Beta-PR15048-3858efd.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18357/HighFidelity-Beta-PR15048-b98eda4.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18357/HighFidelity-Beta-Interface-PR15048-b98eda4.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18357/HighFidelity-Beta-PR15048-b98eda4.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18357/HighFidelity-Quest-Beta-PR15048-b98eda4.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18357/HighFidelity-Beta-PR15048-b98eda4.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18357/HighFidelity-Beta-Interface-PR15048-b98eda4.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18378/HighFidelity-Beta-PR15048-9a8c5e5.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18378/HighFidelity-Quest-Beta-PR15048-9a8c5e5.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18378/HighFidelity-Beta-PR15048-9a8c5e5.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18378/HighFidelity-Beta-Interface-PR15048-9a8c5e5.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18378/HighFidelity-Beta-PR15048-9a8c5e5.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18378/HighFidelity-Beta-Interface-PR15048-9a8c5e5.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18392/HighFidelity-Beta-PR15048-060932a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18392/HighFidelity-Beta-Interface-PR15048-060932a.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18392/HighFidelity-Beta-PR15048-060932a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18392/HighFidelity-Beta-Interface-PR15048-060932a.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18392/HighFidelity-Beta-PR15048-060932a.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18392/HighFidelity-Quest-Beta-PR15048-060932a.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18435/HighFidelity-Beta-PR15048-f3708ba.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18435/HighFidelity-Beta-Interface-PR15048-f3708ba.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18435/HighFidelity-Beta-PR15048-f3708ba.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18435/HighFidelity-Quest-Beta-PR15048-f3708ba.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18435/HighFidelity-Beta-PR15048-f3708ba.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18435/HighFidelity-Beta-Interface-PR15048-f3708ba.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18450/HighFidelity-Beta-PR15048-c3e5c49.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18450/HighFidelity-Beta-Interface-PR15048-c3e5c49.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18450/HighFidelity-Beta-PR15048-c3e5c49.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18450/HighFidelity-Quest-Beta-PR15048-c3e5c49.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18450/HighFidelity-Beta-PR15048-c3e5c49.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18450/HighFidelity-Beta-Interface-PR15048-c3e5c49.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18455/HighFidelity-Beta-PR15048-8627a92.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18455/HighFidelity-Beta-Interface-PR15048-8627a92.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18455/HighFidelity-Beta-PR15048-8627a92.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18455/HighFidelity-Beta-Interface-PR15048-8627a92.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18455/HighFidelity-Beta-PR15048-8627a92.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18455/HighFidelity-Quest-Beta-PR15048-8627a92.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18468/HighFidelity-Beta-PR15048-3241d2f.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18468/HighFidelity-Beta-Interface-PR15048-3241d2f.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18468/HighFidelity-Beta-PR15048-3241d2f.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18468/HighFidelity-Quest-Beta-PR15048-3241d2f.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18468/HighFidelity-Beta-PR15048-3241d2f.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18468/HighFidelity-Beta-Interface-PR15048-3241d2f.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18482/HighFidelity-Beta-PR15048-2622408.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18482/HighFidelity-Quest-Beta-PR15048-2622408.apk)

**Comment by wayne-chen** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18485/HighFidelity-Beta-PR15048-2622408.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18485/HighFidelity-Beta-Interface-PR15048-2622408.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18485/HighFidelity-Beta-PR15048-2622408.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18485/HighFidelity-Quest-Beta-PR15048-2622408.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18485/HighFidelity-Beta-PR15048-2622408.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18485/HighFidelity-Beta-Interface-PR15048-2622408.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18494/HighFidelity-Beta-PR15048-426ffe5.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18494/HighFidelity-Quest-Beta-PR15048-426ffe5.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18494/HighFidelity-Beta-PR15048-426ffe5.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18494/HighFidelity-Beta-Interface-PR15048-426ffe5.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18494/HighFidelity-Beta-PR15048-426ffe5.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18494/HighFidelity-Beta-Interface-PR15048-426ffe5.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18502/HighFidelity-Beta-PR15048-c4cccc6.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18502/HighFidelity-Beta-Interface-PR15048-c4cccc6.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18502/HighFidelity-Beta-PR15048-c4cccc6.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18502/HighFidelity-Quest-Beta-PR15048-c4cccc6.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18502/HighFidelity-Beta-PR15048-c4cccc6.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18502/HighFidelity-Beta-Interface-PR15048-c4cccc6.exe

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -1291,6 +1295,21 @@ Application::Application(int& argc, char** argv, QElapsedTimer& startupTimer, bo
         setCrashAnnotation("display_plugin", displayPlugin->getName().toStdString());
         setCrashAnnotation("hmd", displayPlugin->isHmd() ? "1" : "0");
     });
+    connect(this, &Application::activeDisplayPluginChanged, this, [&](){
+#if !defined(Q_OS_ANDROID)
+        if (!getLoginDialogPoppedUp() && _desktopRootItemCreated) {
+/*            if (isHMDMode()) {*/
```
[the remark this answers is not in the document]
Comment (wayne-chen) [reply interface/src/Application.cpp]: removed

**Pull request merged**